test/functional/DDR_Test/playlist.xml

```diff
@@ -40,6 +40,25 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		</impls>
 	</test>
 
+	<test>
+		<testCaseName>testDDRExt_Class</testCaseName>
+		<command>ant -DJAVA_COMMAND=$(JAVA_COMMAND) -DTEST_ROOT=${TEST_ROOT} -DTEST_JDK_HOME=${TEST_JDK_HOME} -DJDK_VERSION=${JDK_VERSION} \
+	-DTEST_RESROOT=$(TEST_RESROOT) -DRESOURCES_DIR=${RESOURCES_DIR} -DREPORTDIR=${REPORTDIR} -DOS=${OS} -DBITS=$(BITS) -DLIB_DIR=${LIB_DIR} \
+	-Dtest.list=$(Q)TestClassExt$(Q) -DADDITIONALEXPORTS=$(ADDEXPORTS_JDKASM_UNNAMED) -f $(Q)$(TEST_RESROOT)$(D)tck_ddrext.xml$(Q); \
+	$(TEST_STATUS)</command>
+		<platformRequirements>^os.win</platformRequirements>
```

Comment:
It should also run on windows. See VM_Sanity.csv

Reply:
shall I make another test target only for os.win with level sanity? Or shall I simply delete this platform requirement ^os.win ?

Reply:
We can just remove the platformRequirements as all DDR tests in playlist.xml run in extended.